README.md

```diff
@@ -82,6 +82,20 @@ See the [Reporter Interface section of the API documentation](https://github.com
 
 Looking for more examples? Check out the [examples folder](https://github.com/hapijs/good/tree/master/examples).
 
+## Reporter Type
```

Comment:
This PR is just for documentation and we don't need the same thing in both places. That's just something that needs maintained in two places.